docs/concepts/07_dip/05_dapp_developer.md

```diff
@@ -26,8 +26,128 @@ Import the SDK into your code:
 import { * } from '@kiltprotocol/dip-sdk'
 ```
 
+## Example application
+
+This example application walks through the code you can find in the tests for the DIP SDK.
+
+### 1. Generate a base proof
+
+Start by generating the base proof for the DIP using the `[generateDipSiblingBaseProof](https://kiltprotocol.github.io/dip-sdk/functions/generateDipSiblingBaseProof.html)` method and passing the desired configuration:
+
+```typescript
+const baseDipProof = await DipSdk.generateDipSiblingBaseProof(config)
+```
+
+:::info What's a base proof?
+A base proof is a cross-chain state proof, revealing the parts of a DID stored on the KILT blockchain.
+:::
+
+The configuration for the base proof takes the following parameters:
+
+-   `DipSiblingBaseProofInput`: The input for the base proof. It can be one of the following values:
+    -   `didUri`: (Required) The DID URI of the DIP subject performing the cross-chain operation.
+    -   `keyIds`: (Required) An array of verification method IDs of the DID revealed in the cross-chain operation.
```

Comment:
E.g., `#key-1`, `#key-2`, etc.

Reply:
@ntn-x2 Sure. Do you think we need to show that, though? Especially just in the form of "key 1", "key 2", etc

Reply:
Yeah ok maybe not.